### DISCUSSION TITLE
v2rayA updates

### DISCUSSION THREAD

**Discussion by malikshi** [helmiau/HelmiWrt-OS#33] [Ideas]
Bisa ga mas di `vasm` nya untuk update pakai latest release commit direpo v2rayA ada workflow tiap ada commit langsung compile/release hasil commit nya. Cek [V2rayA](https://github.com/v2rayA/v2rayA/actions/workflows/release.yml)

**Comment by helmiau** [comment]
cek cek dlu mas

**Reply by malikshi** [reply]
mas konfirmasi di fw ini udah ke install git blom? soalnya ada yg pke commat git buat stripping nya

**Reply by helmiau** [reply]
> mas konfirmasi di fw ini udah ke install git blom? soalnya ada yg pke commat git buat stripping nya

udah mas

**Reply by malikshi** [reply]
udah bisa mas scriptnya.
cuman pke [nightly.link](https://nightly.link/v2rayA/v2rayA/workflows/release/master)

**Reply by helmiau** [reply]
oke siap

**Reply by malikshi** [reply]
sent PR #35

**Comment by malikshi** [comment]
Tambahan, misal v2rayA sedang running dan kita jalanin install update v2rayA binarynya kemungkinan bakal error. Add checker kalau v2rayA sudah running ngrep dari port atau ps yang berjalan bisa mas, dan stop v2rayA jika v2raya running. + Add notice kalau install/update untuk pakai koneksi dengan kouta/inject tools lain supaya tidak error?

**Reply by helmiau** [reply]
spertinya ga bakal sy tambah mas, karena kasihan sama device yg kentang, nanti malah turun performa dan ganggu koneksi

**Reply by malikshi** [reply]
Oke mas.

**Reply by helmiau** [reply]
dah sy tambah
- cek v2raya run sblm install
- tulisan "mohon gunakan kuota reguler untuk install/update"

**Reply by malikshi** [reply]
oke makasih mas.
